enclave-runner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "enclave-runner"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
Is this enough, or do we need a minor version bump?

Reply:
I guess this is technically a breaking change, yes.